nengo_gui/examples/tutorial/25-spa-parse.py

```diff
@@ -0,0 +1,26 @@
+# Tutorial 25: Parsing Simple Commands
```

Comment:
In theory I think this is the functionality:  If you are shown a command to SAY or to WRITE -> then store that as a verb in memory e.g., if you are shown SAY, store SAY*VERB*0.4 in memory. Similarly if you are shown either A,B,C or D -> then store it as a noun in memory. If you are not shown any of the above, and the verb in memory is either WRITE or SAY, then depending on whether it is WRITE or SAY, either write or speak the noun stored in memory. 

But it doesn't quite work for me when I run the model. This might be dependant on the 0.4 factor that is being used (specially because the same memory is being used to store both the verb and the noun).